### PR TITLE
feat: move to ubuntu mantic (noml)

### PR DESCRIPTION
**Pull request by martabal**
See #264

### CODE REVIEW

**Comment by martabal** [comment]
Damn, 6 hours to build it ?

**Review by hydazz — APPROVED** [review]

**Comment by hydazz** [comment]
+ update the changelog in readme

**Comment by martabal** [comment]
Oh, on mobile, the merge button is on top, literally saw your message just after hitting the merge button 🤦🏻‍♂️

**Pull request merged**
